### ISSUE TITLE
Importing multiple pdfs from nested directories: Not all pdfs get OCR'd

### ISSUE THREAD

**Issue by Moini** [openpaperwork/paperwork#722]
I've tried to import my document directories (many nested directories, well sorted, hundreds of pdfs).
The documents seem to be imported (available only after a restart of the app, though, see https://github.com/openpaperwork/paperwork/issues/721), but most of them do not have their contents OCR'd and indexed.

I wasn't able to find a rule that explains it - the issue here being that file names and directory structure are not accessible from inside the application - it's difficult to find which document corresponds to which file on my disk.

This is with paperwork 1.2.2 (flatpak) on Linux Mint 18.3.

**Comment by jflesch** [comment]
By default, Paperwork runs OCR only on PDF pages that don't contain text at all. If your PDF already had text, OCR is not used.
You can however use the option "Redo the OCR" if required.

**Comment by Moini** [comment]
Yes. None of my pdfs contain any text, because they've only been converted from jpg to pdf by convert.  
I've tried 'Redo OCR' for all documents via settings, and even then, it didn't work. I'm still going from document to document currently, checking for blue boxes (activated via view menu item), and redoing OCR for each document separately. That mostly works (except for bad scans, which is to be expected).

**Comment by jflesch** [comment]
I have a feeling that all your tickets (#721, #722, #723, etc) stem from the same bug. I think the diagnostic output could really help figure out what's going on here.

**Comment by Moini** [comment]
Yeah, maybe. Paperwork also sometimes, after a couple of clicks, or back and forth between documents, starts using one full CPU and lots of memory, even if I'm doing nothing. I'll try to get some diagnostics for you.

**Comment by jflesch** [comment]
I think it's clearly a duplicate of #721
